template-procedure.md

```diff
@@ -0,0 +1,251 @@
+
+Write a few sentences to describe the task and provide additional context on the task.
+
+[Task]:
+
+1. Step 1
+2. Step 2
+
+
+## Procedure Content Samples
+
+The following are common content types that may appear in procedural topics.
+Copy and paste the markdown to use it in your topic.
+
+
+### “Fill-in-the-Fields” Table
+
+Where the reader must enter many values in, say, a YAML file, use a table within the procedure as
+below:
+Open the YAML file.
+
+```yaml
+Key1: Value1
+Key2: Value2
+metadata:
+  annotations:
+    # case-sensitive
+    Key3: Value3
+  Key4: Value4
+  Key5: Value5
+spec:
+  # Configuration specific to this broker.
+  config:
+    Key6: Value6
+```
+
+Change the relevant values to your needs, using the table below as a guide.
+
+| Key   | Value Type     | Description  |
+|-------|----------------|--------------|
+| Key1  | String         | Description  |
+| Key2  | Integer        | Description  |
+| Key3  | String         | Description  |
+| Key4  | String         | Description  |
+| Key5  | Float          | Description  |
+| Key6  | String         | Description  |
+
+
+### Table
+
+Introduce the table with a sentence. For example, “The table below details which operations must be
+made available to a developer accessing a Knative Route using a minimal profile.”
+
+
+#### Markdown Table Template
+
+|Header 1|Header 2|
+|--------|--------|
+|Data1   |Data2   |
+|Data3   |Data4   |
+
+
+### Ordered List
+
+Write a sentence or two to introduce the content of the list.
+For example, “If you want to fix or add content to a past release, you can find the source files in
+the following folders.”. Optionally, include bold lead-ins before each list item.
+
+
+#### Markdown Ordered List Templates
+
+1. Item 1
+1. Item 2
+1. Item 3
+
+1. **Lead-in description:** Item 1
+1. **Lead-in description:** Item 2
+1. **Lead-in description:** Item 3
+
+
+### Unordered List
+
+Write a sentence or two to introduce the content of the list.
+For example, “Your own path to becoming a Knative contributor can begin in any of the following
+components:”. Optionally, include bold lead-ins before each list item.
+
+
+#### Markdown Unordered List Template
+
+* List item
+* List item
+* List item
+
+* **Lead-in**: List item
+* **Lead-in**: List item
+* **Lead-in**: List item
+
+
+#### Note
+
+Ensure the text beneath the **note** is indented as much as **note** is.
+
+!!! note
+    This is a note.
+
+
+#### Warning
+
+If the note regards an issue that could lead to data loss, the note should be a warning.
+
+!!! warning
+    This is a warning.
+
+
+#### Markdown Embedded Image
+
+The following is an embedded image reference in markdown.
+
+![Annotated procedure topic](../image/example-reference-topic.png)
+
+
+#### Tabs
+
+Place multiple versions of the same procedure (such as different versions for different products)
+within tabs. Indent the opening tabs tags 3 spaces to make the tabs display properly.
+
+   {{< tabs name="name" default="name" >}}
```

Comment:
The formatting for tabs has changed since the move to MKdocs. I've found that the content must be indented 4 spaces relative to the indentation for the `=== "tab name"` line, otherwise the tabs will break in various interesting ways. Also, there is no closing tag.

Formatting is as follows:

```
=== "tab1 name"

    Tab content

=== "tab2 name"

    Tab content
```

Reply:
Thanks, well caught.